README.md

```diff
@@ -74,6 +86,7 @@ List of options provided by CMake:
 | Name | Description | Values | Default |
 | - | - | - | - |
 | UMF_BUILD_SHARED_LIBRARY | Build UMF as shared library | ON/OFF | OFF |
+| UMF_BUILD_LIBPOOL_JEMALLOC | Build the libpool_jemalloc static library | ON/OFF | ON |
```

Comment:
It should be set to OFF by default as it requires dependencies.

Reply:
Done